app/imports/import_validators.go

```diff
@@ -312,6 +316,26 @@ func ValidateUserImportData(data *UserImportData) *model.AppError {
 	return nil
 }
 
+var validAuthServices = []string{"", "email", "gitlab", "saml", "ldap", "google", "office365"}
+
+func validateAuthService(authService *string, cloud bool) *model.AppError {
+	services := validAuthServices
+	if cloud {
+		services = services[:3]
+	}
```

Comment:
What if we do this instead of the `[:3]` thing?
```suggestion
var cloudValidAuthServices = []string{"", "email", "gitlab"}
var allValidAuthServices = append(cloudValidAuthServices, "saml", "ldap", "google", "office365")

func validateAuthService(authService *string, cloud bool) *model.AppError {
	services := allValidAuthServices
	if cloud {
		services = cloudValidAuthServices
	}
```

Reply:
Oh wait. That's not supposed to be in the PR yet. I accidentally committed parts of the next PR.

Reply:
:thinking:

Reply:
:facepalm:

Reply:
Ah, I see now :joy: